assets/src/stories-editor/components/media-inserter/index.js

```diff
@@ -21,21 +21,35 @@ const POPOVER_PROPS = {
 	position: 'bottom right',
 };
 
-const MediaInserter = ( { insertBlock, updateBlock, canInsertBlockType, showInserter } ) => {
+const MediaInserter = ( { insertBlock, updateBlock, canInsertBlockType, showInserter, getCurrentPageAttributes } ) => {
 	const blocks = [
 		'core/video',
 		'core/image',
 	];
+	const currentBlock = getCurrentPageAttributes();
+
+	let imageTitle = __( 'Insert Background Image', 'amp' );
+	let videoTitle = __( 'Insert Background Video', 'amp' );
+	if ( currentBlock.mediaId ) {
+		switch ( currentBlock.mediaType ) {
```

Comment:
ESLint complains about a missing default case.

I'd probably just write something like this anyway:

```js
const imageTitle = 'image' === currentBlock.mediaType ? __( 'Update Background Image', 'amp' ) : __( 'Insert Background Image', 'amp' );
const videoTitle = 'video' === currentBlock.mediaType ? __( 'Update Background Video', 'amp' ) : __( 'Insert Background Video', 'amp' );
```

Reply:
Resolved in https://github.com/ampproject/amp-wp/pull/3109/commits/ba5a34c8189deeeede2de789246adbde189d7886